sdk/src/client/stronghold/mod.rs

```diff
@@ -286,7 +292,9 @@ impl StrongholdAdapter {
     /// data, provide a list of keys in `keys_to_re_encrypt`, as we have no way to list and iterate over every
     /// key-value in the Stronghold store - we'll attempt on the ones provided instead. Set it to `None` to skip
     /// re-encryption.
-    pub async fn change_password(&self, new_password: &str) -> Result<(), Error> {
+    pub async fn change_password(&self, new_password: impl Into<Password> + Send) -> Result<(), Error> {
+        let new_password = new_password.into();
```

Comment:
Also put this here?
```suggestion
        // Note: we deliberately put the password conversion right here. Only change if you know what you're doing.
        let new_password = new_password.into();
```

Reply:
I'd rather just remove these comments. They aren't helpful.

Reply:
Also fine for me to remove everywhere